events/galaxy-academy-2024.md

```diff
@@ -1,54 +1,135 @@
 ---
-layout: learning-pathway
+layout: event
 
 title: Galaxy Academy 2024
 description: |
-  The Galaxy Academy is a online training event for Beginners as well as learners who would like to improve there Galaxy data analysis skills. Over the course of one week we will have a different topic and focus every day. 
-tags: [beginner]
+  The Galaxy Academy is a online training event for Beginners as well as learners who would like to improve their Galaxy data analysis skills. Over the course of one week, we will have a different topic and focus every day. 
+
 draft: true
 
+date_start: 2024-10-07
+date_end: 2024-10-11
+
```

Comment:
Don't forget to add the registration info

```
registration:
  link: https://example.org
  deadline: 2024-01-01
```

Reply:
Can we add this after we go life? I think the form is not ready yet.

Reply:
sure, I will quickly write a way to indicate registration will open in the future, so that the test doesn't fail on this